test/bigdecimal/test_bigdecimal.rb

```diff
@@ -181,6 +181,7 @@ def test_BigDecimal_with_float
     end
     BigDecimal.save_exception_mode do
       BigDecimal.mode(BigDecimal::EXCEPTION_INFINITY, false)
+      p ahi: Float::INFINITY, uhi: Float::INFINITY.infinite?
```

Comment:
looks like you left a debugging statement?

Reply:
It's intuitive insertion. I'm working on debugging [the issue occurred on mswin environment](https://github.com/ruby/bigdecimal/runs/6088387291?check_suite_focus=true).